src/aws/aws-sdk.js

```diff
@@ -0,0 +1,29 @@
+const AWS = require("aws-sdk");
```

Comment:
rename this file... dont put in a separate aws folder. instead have it in utils

Reply:
added in utils